components/brave_ads/core/database.h

```diff
@@ -35,6 +35,8 @@ class ADS_EXPORT Database final {
                       mojom::DBCommandResponseInfo* command_response);
 
  private:
+  bool IsValid();
```

Comment:
const

Reply:
cannot make this function const because `db_.GetUniqueStatement()` which is called inside is not const